sdk/cosmosdb/cosmos/tsconfig.src.build.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../tsconfig.src.build.json",
```

Comment:
@deyaaeldeen Could omitting `"composite": true` from the `tsconfig.src.build.json` (present in `tsconfig.src.json`) config cause any regressions?